cli/package.json

```diff
@@ -185,6 +186,9 @@
           "{projectRoot}/build"
         ]
       }
-    }
+    },
+    "implicitDependencies": [
```

Comment:
This tells Nx that all of the NPM packages need to be built before the CLI is built. We need to do this because the post-build script in CLI copies the built assets for each NPM package into the `cli/build` folder, so when that happens, all of the packages need to have already been built.